### DISCUSSION TITLE
cannot generate rice-box.go file

### DISCUSSION THREAD

**Discussion by JasonYjz** [filebrowser/filebrowser#1532] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/37281108/131621730-ef813294-3c70-49fb-abfe-4b6d12753430.png)

According to this above guide, after executing the **both 'cd http' and 'rice embed-go' command,** the 'rice-box.go' file is not generated under this 'http' folder.

Is there any steps missing? or how to resolve this issue.
I'm blocked here now, pls give some help for this. Thanks.

**Comment by JasonYjz** [comment]
and it output the following message after executing the 'rice embed-go' command:

root@aid_test: /http# rice embed-go
_2021/09/01 06:06:54 no calls to rice.FindBox() found_

**Comment by ramiresviana** [comment, marked as answer]
The contributing guide is outdated, currently Filebrowser does not make use of go.rice, it will automatically embed the assets using embed.FS.

**Reply by JasonYjz** [reply]
After through the souce code, found that directly parse this 'frontend/dist' folder.
But after starting the Filebrowser server, still cannot see any frontend page view by accessing to the url like(xxx.xxx.xxx.xxx:8080). There is one certain point that the frontend package is existed in this 'dist' folder.

I'm completely new to the Filebrowser. Now I just want to see the view through the chrome browser, but still failed, also I don't know any other extra steps need to configure. 
Could you sychronized update the guide.

**Reply by JasonYjz** [reply]
Today, i rebuild the frontend pakcage and backend service. Now i can see the view from the chrome browser. 
Exciting so much. 

Still thanks for your reply. One suggestion here, better to sychronzied update the guide, or it will be confusing to new hand, like me. ^_^

**Reply by alexisLefebvre** [reply]
The documentation is here: https://github.com/filebrowser/docs/blob/master/contributing/README.md
